### PR TITLE
GH actions

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Point of discussion: Should we set action version refs to `master` or remove them?

### CODE REVIEW

File: ci/packages.sh [hunk 1]
```diff
@@ -73,8 +79,20 @@ done
...
+PACKAGES_LIST="["
+for PKG in $PACKAGES
+do
+    if [[ "${PACKAGES_LIST}" != "[" ]]
+    then
+        PACKAGES_LIST+=", "
+    fi
+    PACKAGES_LIST+="'${PKG}'"
+done
+PACKAGES_LIST+="]"
```
Comment (ar13pit) [inline ci/packages.sh:91]:
```suggestion
for PKG in $PACKAGES
do
    PACKAGES_LIST="${PACKAGES_LIST:+$PACKAGES_LIST, }'${PKG}'"
done
PACKAGES_LIST="[${PACKAGES_LIST}]"
```

File: ci/commit-range/action.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        then
+          OLDEST=${{ github.event.pull_request.base.sha }}
+          NEWEST=${{ github.event.pull_request.head.sha }}
+          COMMIT_RANGE="${OLDEST}...${NEWEST}"
+        fi
+        echo -e "\e[35m\e[1m COMMIT_RANGE = ${COMMIT_RANGE} \e[0m"
+        echo "::set-output name=commit-range::${COMMIT_RANGE}"
+      shell: bash
+
+
```
Comment (ar13pit) [inline ci/commit-range/action.yml:34]:
```suggestion
```

**Review by ar13pit — CHANGES_REQUESTED** [review]
Except for the suggested changes this LGTM

File: ci/modified-packages/action.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      id: packages
+      run: |
+        PULLREQUEST=${{ github.event.number }}
+        PULLREQUEST=${PULLREQUEST:-false}
+        source ${{ github.action_path }}/../packages.sh --pullrequest=${PULLREQUEST} --branch=${GITHUB_BASE_REF} --commit-range=${{ inputs.commit-range }} --all=${{ github.event_name == 'schedule' }} ${{ inputs.ignored-packages }}
+        echo "::set-output name=packages::${PACKAGES_LIST}"
+      shell: bash --noprofile --norc -o pipefail {0}
+
+
+
```
Comment (ar13pit) [inline ci/modified-packages/action.yml:35]:
```suggestion
```

Comment (MatthijsBurgh) [inline ci/modified-packages/action.yml]:
```suggestion
```

**Review by ar13pit — APPROVED** [review]

**Review by ar13pit — DISMISSED** [review]

**Review by ar13pit — APPROVED** [review]

**Pull request merged**